### PR TITLE
fix: unify link components

### LINKED ISSUE #3633: unify Link components
currently there are 4 different approaches in the code regarding usage of external links:

[@trezor/components/Link](https://github.com/trezor/trezor-suite/blob/develop/packages/components/src/components/typography/Link/index.tsx) - high level component used for external links it's covering cases like `noreferrer noopener` or `stopPropagation`

[TrezorLink](https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/components/suite/TrezorLink/index.tsx) - for handling TOR links, depends on `@trezor/components/Link`

[ExternalLink](https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/components/suite/ExternalLink/index.tsx) - `TrezorLink` with 2 predefined props

- [X] `styled.a` - (11 occurrences) not covering anything

- [ ] it would be nice to reduce it to 2 (?)

### PR DESCRIPTION
**Pull request by karliatto**
Related to https://github.com/trezor/trezor-suite/issues/3633
I does not refactor all the links components but some of them.

### CODE REVIEW

File: packages/suite/src/views/dashboard/components/NewsFeed/index.tsx [hunk 1]
```diff
@@ -133,11 +131,11 @@ const NewsFeed = ({ maxVisibleCount = 9 }: Props) => {
         <Section
             heading={<Translation id="TR_WHATS_NEW" />}
             actions={
-                <MediumLink target="_blank" rel="noopener noreferrer" href={mediumUrl}>
+                <TrezorLink target="_blank" href={mediumUrl}>
```
Comment (szymonlesisz) [inline packages/suite/src/views/dashboard/components/NewsFeed/index.tsx]:
```suggestion
                <TrezorLink thref={mediumUrl}>
```
TrezorLink > Link have `target="_blank"` as default

File: packages/suite/src/views/settings/device/index.tsx [hunk 2]
```diff
@@ -254,9 +252,8 @@ const Settings = () => {
                                     values={{
                                         version: (
                                             <VersionTooltip content={revision} disabled={!revision}>
-                                                <VersionLink
+                                                <TrezorLink
```
Comment (szymonlesisz) [inline packages/suite/src/views/settings/device/index.tsx]: remove "target" as well

Comment (karliatto) [reply packages/suite/src/views/settings/device/index.tsx]: Done, thanks!

File: packages/suite/src/views/dashboard/components/NewsFeed/index.tsx (hunk 1 above)
Comment (karliatto) [reply packages/suite/src/views/dashboard/components/NewsFeed/index.tsx]: You are totally right! 
Done, thanks!

**Comment by prusnak** [comment]
Why do you remove `rel="noopener noreferrer"`? It's important to keep them for the web version, no?

**Comment by karliatto** [comment]
> Why do you remove `rel="noopener noreferrer"`? It's important to keep them for the web version, no?

Because we changed `MediumLink` for generic `TrezorLink` component and the parent of `TrezorLink` already includes `rel="noopener noreferrer"` https://github.com/trezor/trezor-suite/blob/2b45be33e22df90315580dca14c5f2fed2504061/packages/components/src/components/typography/Link/index.tsx#L70

So it is there but from the parent component.

**Comment by prusnak** [comment]
> So it is there but from the parent component.

Great, thanks!

**Review by szymonlesisz — APPROVED** [review]

File: packages/suite/src/views/dashboard/components/NewsFeed/index.tsx [hunk 3]
```diff
@@ -90,7 +88,7 @@ const PostComponent = (props: PostComponentProps) => {
     const thumbnail = useExternalLink(props.thumbnail);
 
     return (
-        <Post target="_blank" rel="noopener noreferrer" href={url} data-test={props['data-test']}>
+        <Post target="_blank" href={url} data-test={props['data-test']}>
```
Comment (szymonlesisz) [inline packages/suite/src/views/dashboard/components/NewsFeed/index.tsx]:
```suggestion
        <Post href={url} data-test={props['data-test']}>
```
missed that

Comment (karliatto) [reply packages/suite/src/views/dashboard/components/NewsFeed/index.tsx]: Done. Thanks!

**Review by tomasklim — APPROVED** [review]

**Pull request merged**